pages/index.tsx

```diff
@@ -164,3 +166,11 @@ const Home: React.FC<HomeProps> = () => {
 };
 
 export default Home;
+
+export async function getStaticProps() {
+  const { data, error } = await supabase.from("config").select("eventName");
```

Comment:
Avoid database related tasks in frontend as general practice

Reply:
Understood.